### ISSUE TITLE
Use Calibre rather than kindlegen

### ISSUE THREAD

**Issue by calebhearth** [thoughtbot/paperback#93]
Kindlegen has a restrictive license that only allows generated files to be distributed through the Kindle store, meaning that our current process of distributing them via github, upcase, and now gumroad violates the terms of that license.

Discussion of this in #books started [here](https://thoughtbot.slack.com/archives/books/p1410817103000242) and continued for a couple pages, if you'd like to catch up.

Basically, Calibre would generate mobi files that could be manually transferred to Kindles, but not emailed. There's no solution I'm aware of to generate files that could be emailed to Kindles that wouldn't violate the terms of service or be distributed through the kindle store.
